### ISSUE TITLE
[a11y] Cannot create a new Taxon

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1235]
**Describe the bug**
I cannot create a new Taxon in the fish database after giving it a name and associating it with an existing taxon record.

**To Reproduce**
1. Go to the Taxon form
2. Attempt to create a Taxon

**Screenshots**
![image](https://user-images.githubusercontent.com/37256050/163512877-b6ab3082-8cf2-48e4-b07d-3bc221cc93dd.png)

**Comment by grantfitzsimmons** [comment]
I cannot create a new Geography either. If this is a separate issue I can make a new bug:

<img width="897" alt="image" src="https://user-images.githubusercontent.com/37256050/163513495-e7a65ad0-81de-4102-bc10-e3458a68439b.png">

No definition item

**Comment by maxxxxxdlp** [comment]
Does this error happen if you choose a parent and a rank?
Those files should be required. The fact that they aren't is the bug

**Comment by grantfitzsimmons** [comment]
![image](https://user-images.githubusercontent.com/37256050/163576873-5133b20e-5634-4856-b722-6551c54ff5d4.png)

Rank is bugged, you are correct!

**Comment by maxxxxxdlp** [comment]
Fixed in [88f0ebb](https://github.com/specify/specify7/commit/88f0ebbbf4b34e524249073920287fbd75d25aa1)

**Comment by grantfitzsimmons** [comment]
![image](https://user-images.githubusercontent.com/37256050/163586997-628a5479-6aee-4124-877e-660c7c68c215.png)

Same issue in Storage tree. I will test when changes are live
